src/ncp-spinel/SpinelNCPInstance.cpp

```diff
@@ -2039,6 +2039,30 @@ SpinelNCPInstance::property_set_value(
 				);
 			}
 
+		} else if (strcaseequal(key.c_str(), kWPANTUNDProperty_IPv6MeshLocalPrefix)) {
+			struct in6_addr address = any_to_ipv6(value);
+			char address_str[INET6_ADDRSTRLEN] = "::";
+
+			inet_ntop(AF_INET6, (const void *)&address, address_str, sizeof(address_str));
+
+			fprintf(stderr, "Using prefix \"%s\"\n", address_str);
+
+			Data command =
+				SpinelPackData(
+					SPINEL_FRAME_PACK_CMD_PROP_VALUE_SET(SPINEL_DATATYPE_DATA_S),
+					SPINEL_PROP_IPV6_ML_PREFIX,
+					address.s6_addr,
+					8
+				);
+
+			mSettings[kWPANTUNDProperty_IPv6MeshLocalPrefix] = SettingsEntry(command);
```

Comment:
I am not sure if want to save the Mesh-Local Prefix in the `mSettings`?
This is not done for channel, PANId, XPANID, NetworkName, ...and other similar network info.

NCP/OT already saves the mesh-local prefix as part of Operational Dataset (in flash in its settings).  The `mSettings` in `wpantund` is intended for configurations settings (e.g., jamming detection settings) that we need to persist over NCP resets but the are not saved by the OT/NCP itself.

Reply:
Updated. Thanks for the explanations!